### PR TITLE
Managing sub

### PR DESCRIPTION
**Pull request by idalithb**
-Separating two pages

### CODE REVIEW

File: nginx.conf [hunk 1]
```diff
@@ -94,7 +94,8 @@ http {
         rewrite ^/docs/([a-zA-Z][a-zA-Z])/managing/transferring-subgraph-ownership/$    $scheme://$http_host/docs/$1/managing/transfer-and-deprecate-a-subgraph/    permanent;
         rewrite ^/docs/en/substreams/(?!index\.).+$                                     https://substreams.streamingfast.io                                         permanent;
         rewrite ^/docs/en/firehose/(?!index\.).+$                                       https://firehose.streamingfast.io                                           permanent;
-
+        rewrite ^/docs/([a-zA-Z][a-zA-Z])/managing/transfer-and-deprecate-a-subgraph/$  $scheme://$http_host/docs/$1/managing/transfer-a-subgraph/                  permanent;
+        rewrite ^/docs/([a-zA-Z][a-zA-Z])/managing/transfer-and-deprecate-a-subgraph/$  $scheme://$http_host/docs/$1/managing/deprecate-a-subgraph/                 permanent;
```
Comment (benface) [inline nginx.conf:98]: Looks like you're attempting to redirect the same URL (`/docs/[locale]/managing/transfer-and-deprecate-a-subgraph/`) to multiple URLs. That is not possible; only the first `rewrite` rule will take effect.

File: website/pages/ar/managing/deprecate-a-subgraph.mdx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
+title: Deprecate a Subgraph
+---
+
+## Deprecating a Subgraph
+
+Although you cannot delete a subgraph, you can deprecate it on Graph Explorer.
+
+### Step-by-Step
```
Comment (benface) [inline website/pages/ar/managing/deprecate-a-subgraph.mdx:9]:
```suggestion
Although you cannot delete a subgraph, you can deprecate it on Graph Explorer.

## Step-by-Step
```

Comment (benface) [reply website/pages/ar/managing/deprecate-a-subgraph.mdx:9]: Oops, I just realized I left this comment for the file in the `ar` locale... well, I guess it applies to all of them. 😬

File: nginx.conf (hunk 1 above)
Comment (idalithb) [reply nginx.conf:98]: @benface I need a little clarity for future redirects (lets chat)

File: nginx.conf
Comment (benface) [inline nginx.conf:1]: @idalithb – Redirects at lines 93 and 94 should be removed.

**Review by benface — APPROVED** [review]
Great job!

**Pull request merged**